CentOS_7/job/boot.sh

```diff
@@ -19,15 +19,14 @@ boot() {
 }
 
 wait_for_exit() {
-  ret=0
   is_success=false
 
-  exec_cmd "echo Waiting for container $TASK_CONTAINER_NAME to exit"
+  exec_cmd "echo Waiting for $TASK_CONTAINER_NAME to exit"
```

Comment:
this will show up in the booting container for task group
https://rcapp.shippable.com/github/Bharath92/jobs/wait_for_exit_test_success/builds/5a66e64e49850c0500cb9108/console

Reply:
Okay, looked at the design. Seems like we want to this.